scss.json

```diff
@@ -0,0 +1,19 @@
+{
+  "rules": {
+    "at-rule-no-unknown": null,
+    "scss/at-rule-no-unknown": true,
+    "scss/map-keys-quotes": "always"
+  },
```

Comment:
Wäre es nicht besser die rules auch auf `scss` Dateien zu beschränken? Also in `overrides` zu verschieben

Reply:
Ja, stimmt! 👍